.github/workflows/changeset.yaml

```diff
@@ -5,6 +5,18 @@ on:
     branches:
       - "main"
 
+  # When Changesets opens a PR it does not trigger GitHub actions,
+  # because its token does not have permission to. This is a hack
+  # to allow one of us to trigger GitHub actions for a changesets PR
+  # by enabling automerge on the PR.
+  pull_request_target:
```

Comment:
I think you might need a new workflow file, because currently it looks like this workflow only runs _on_ `main`.

You'll need another one which runs on the `changeset-release/main` branch instead.

Reply:
Oh you're right, they use it on their CI which monitors all branches